output/xml/default.xml

```diff
@@ -227,6 +227,7 @@
     <property name="file" type="text"/>
     <property name="gen_file" type="text"/>
     <property name="type" type="text"/>
+    <property name="precompiled_header" type="text"/>
```

Comment:
Can this property be wrapped inside a category `C++ Properties` like it's done in the Project template? These properties are not displayed anywhere currently but it would make the xml more clear that this is a C++ only property.